ui/src/statistics/spurious.js

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { Box, TextField, Typography } from '@mui/material';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { DesktopDatePicker } from '@mui/x-date-pickers/DesktopDatePicker';
+
+import { DataGrid } from '@mui/x-data-grid';
+
+const columns = [
+  { field: 'test_name', headerName: 'Test Name', flex: 1 },
+  { field: 'count', headerName: '# Failures', flex: 1 }
+];
+
+const SpuriousFailuresView = props => {
+  const [value, setValue] = React.useState(new Date());
+
+  const [results, setResults] = React.useState([]);
+
+  const handleChange = newValue => {
+    fetch(`/api/spurious/${newValue.unix()}`)
+      .then(response => response.json())
+      .then(result => {
+        setResults(result);
+      });
+    setValue(newValue);
+  };
+
+  return (
+    <>
+      <Typography variant="h4">Nightly Spurious Failures</Typography>
+      <LocalizationProvider dateAdapter={AdapterDayjs}>
+        <DesktopDatePicker
+          label="Since Date"
+          inputFormat="DD/MM/YYYY"
+          value={value}
+          onChange={handleChange}
+          disableFuture={true}
+          maxDate={new Date()}
```

Comment:
I think they do the same... (at least while we're just picking dates I think)

Reply:
I thought so too initially, but they don't actually... :confounded: 

One disabled the _option_ to click future dates, while the other hides it.

But since you bring it up, I guess only `max_date` is needed.